packages/caliper-fabric/lib/configValidator.js

```diff
@@ -513,17 +541,40 @@ class ConfigValidator {
         if (result.error) {
             throw result.error;
         }
+        Logger.debug('Exiting validateOrganization');
+    }
+
+    /**
+     * Validates the given organization wallet configuration object.
+     * @param {object} config The configuration object.
+     */
+    static validateOrganizationWallet(config) {
+        Logger.debug('Entering validateOrganizationWallet');
+        const schema = j.object().keys({
+            path: j.string().min(1).optional(),
```

Comment:
If we have an org entry, then path should be required. Otherwise, the following is also valid:
```yaml
organizationWallets:
  org1:
  org2:
    path: 'path'
```